back/db/schema.rb

```diff
@@ -1087,7 +1085,6 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string "logo"
-    t.string "header_bg"
```

Comment:
See [comment](https://github.com/CitizenLabDotCo/citizenlab/pull/2230/files#r960366946) on deleting this from `tenants`.